system/exceptions/Whoops.cfm

```diff
@@ -100,7 +100,9 @@
 			"NativeErrorCode"      : oException.getNativeErrorCode(),
 			"SQL Sent"             : oException.getSQL(),
 			"Driver Error Message" : oException.getqueryError(),
-			"Name-Value Pairs"     : oException.getWhere()
+			"Name-Value Pairs"     : oException.getWhere(),
```

Comment:
is this in both lucee and Adobe?

Reply:
i added the 2 below this code `"Name-Value Pairs" : oException.getWhere(),`
The message for sure, since it gets grabbed lower in the whoops template, the datasource i would assume would be available in both